docs/book/src/development/development.md

```diff
@@ -195,6 +196,113 @@ and create the environment by running:
 ./hack/ci/create_devstack.sh
 ```
 
+##### DevStack
+
+Here's a few notes to setup a DevStack environment and debug ressources (tested on `m3.small` from Equinix Metal: https://deploy.equinix.com/product/servers/m3-small/)
+
+###### Server side
+
+As a root user, install and configure DevStack:
+
+```
+# useradd -s /bin/bash -d /opt/stack -m stack
+# chmod +x /opt/stack
+# echo "stack ALL=(ALL) NOPASSWD: ALL" | tee /etc/sudoers.d/stack
+# sudo -u stack -i
+$ git clone https://opendev.org/openstack/devstack
+$ cd devstack
+$ cat > local.conf <<EOF
+[[local|localrc]]
+ADMIN_PASSWORD=!!! CHANGE ME !!!
+DATABASE_PASSWORD=\$ADMIN_PASSWORD
+RABBIT_PASSWORD=\$ADMIN_PASSWORD
+SERVICE_PASSWORD=\$ADMIN_PASSWORD
+
+GIT_BASE=https://opendev.org
+# Enable Logging
+LOGFILE=$DEST/logs/stack.sh.log
+VERBOSE=True
+LOG_COLOR=True
+enable_service rabbit
```

Comment:
Not needed, it's in the defaults: https://github.com/openstack/devstack/blob/c424a7a299e37004d318107648bb18e157344985/stackrc#L64-L83